modules/wri_project/config/install/core.entity_view_display.node.project_detail.main_content.yml

```diff
@@ -198,6 +204,15 @@ content:
             lb: ''
     weight: 6
     region: buttons
+  field_paragraphs:
+    type: entity_reference_revisions_entity_view
+    label: hidden
+    settings:
+      view_mode: default
+      link: ''
+    third_party_settings: {  }
+    weight: 9
+    region: share
```

Comment:
I wasn't very clear with my comment, but for Projects, we want the share links to appear at the bottom of the `main_content` region, under the body text. It can come out of the header and share sections entirely.

Reply:
OK, I _think_ I have this right, but it was behaving oddly trying to export / save the updated configs.